consumer/rock/guides/README.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Useful Guides for ROCK960
 permalink: /documentation/consumer/rock/guides/
-redirect_from: /documentation/consumer/rock960/guides/
```

Comment:
Did you mean to remove this redirect ?

Reply:
Yes. As noted above, you can only have URLs listed once under redirect_from and it is also present in consumer > rock > rock960 > guides > README.md which seems to be a better destination given that the source URL specifically says rock960 and not just rock.